File: Python/auth/reuse_cookies.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    context = browser.new_context(storage_state="cookies.json")
    page = context.new_page()
    page.goto("https://example.com")
    print(page.content())
    browser.close()

File: NodeJS/auth/reuse_cookies.js
const fs = require('fs');
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const storage = JSON.parse(fs.readFileSync('cookies.json', 'utf8'));
  const context = await browser.newContext({ storageState: storage });
  const page = await context.newPage();

  await page.goto('https://example.com');
  console.log(await page.content());

  await browser.close();
})();
